.gitignore

```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+
```

Comment:
- [ ] Good job adding `node_modules` to your `.gitignore` file :+1:, but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should make sure that test.md is actually ignored by git and is not present in your GitHub repository :100: